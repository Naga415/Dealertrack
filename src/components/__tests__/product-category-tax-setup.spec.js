import React from "react";
import renderer from "react-test-renderer";
import ProductCategoryTaxSetup from "../product-category-tax-setup";

describe("Product Category Tax Setup specs", () => {
  const props = {
    htmlId: "ProductCategoryTaxSetup",
    data: [{ feeType: "test1" }, { feeType: "test2" }],
    actions: {}
  };

  it("renders <ProductCategoryTaxSetup />", () => {
    const component = renderer.create(<ProductCategoryTaxSetup {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
