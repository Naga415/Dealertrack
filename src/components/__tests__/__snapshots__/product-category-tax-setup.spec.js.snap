// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Category Tax Setup specs renders <ProductCategoryTaxSetup /> 1`] = `
<div
  className="row"
  id="edit-tax-product-category-setup"
>
  <div
    className="nav-tabs-content"
  >
    <strong
      className="text"
    >
      Following are the products available.
    </strong>
    <table
      className="table product-category-tax-setup-table"
    >
      <thead>
        <tr>
          <th>
            Category
          </th>
          <th>
            Tax if cap
          </th>
          <th>
            Tax if DIA
          </th>
          <th>
            CRT
          </th>
        </tr>
      </thead>
      <tbody>
        <strong>
           No Products available
        </strong>
      </tbody>
    </table>
  </div>
</div>
`;
